### PR TITLE
intensity-normalization: init at 2.0.1

### PR DESCRIPTION
**Pull request by bcdarwin**
python3Packages.scikit-fuzzy: unstable-2020-10-03 -> unstable-2021-03-31

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [NA] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)  [Note: some binaries raise `ImportError` due to the missing optional dependency `antspy`.]
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -2957,6 +2957,12 @@ with pkgs;
 
   inklecate = callPackage ../development/compilers/inklecate {};
 
+  intensity-normalization = callPackage ../applications/science/biology/intensity-normalization {
+    inherit (python3Packages)
+      buildPythonPackage fetchPypi pythonOlder pytestCheckHook
+      matplotlib nibabel numpy scikit-fuzzy scikitimage scikitlearn scipy statsmodels;
```
Comment (bobby285271) [inline pkgs/top-level/all-packages.nix:2963]:
```suggestion
      matplotlib nibabel numpy scikit-fuzzy scikitimage scikit-learn scipy statsmodels;
```

File: pkgs/applications/science/biology/intensity-normalization/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytestCheckHook
+, matplotlib
+, nibabel
+, numpy
+, scikit-fuzzy
+, scikitimage
+, scikitlearn
```
Comment (bobby285271) [inline pkgs/applications/science/biology/intensity-normalization/default.nix:11]:
```suggestion
, scikit-learn
```

File: pkgs/applications/science/biology/intensity-normalization/default.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  '';
+
+  checkInputs = [ pytestCheckHook ];
+  propagatedBuildInputs = [
+    matplotlib
+    nibabel
+    numpy
+    scikit-fuzzy
+    scikitimage
+    scikitlearn
```
Comment (bobby285271) [inline pkgs/applications/science/biology/intensity-normalization/default.nix:37]:
```suggestion
    scikit-learn
```

**Comment by bcdarwin** [comment]
This package has a Python API, so I'm going to close this and make a new PR with this as a Python package.

**Comment by bcdarwin** [comment]
Now opened as #136487.

**Pull request closed without merging**